src/routes/Home/index.tsx

```diff
@@ -300,30 +314,52 @@ const Home = ({ videoId }: { videoId: string }) => {
             {!loadingSelectedVideo ? (
               <div>
                 {selectedVideo ? (
-                  <div>
-                    <div className="ui embed">
-                      <iframe src={videoSrc} allowFullScreen title="Video player" />
-                    </div>
-                    <div className="ui segment">
-                      <h4 className="ui header">{(selectedVideo as any).snippet.title}</h4>
-                      <p>{(selectedVideo as any).snippet.description}</p>
+                  isLandingPage ? (
+                    <LandingPage user={currentUser ? (currentUser as any).firstName : 'Guest'} />
+                  ) : (
+                    <div>
+                      <div className="ui embed">
+                        <iframe src={videoSrc} allowFullScreen title="Video player" />
+                      </div>
+                      <div className="videodesc">
+                        <h3>{(selectedVideo as any).snippet.title}</h3>
+
+                        <p>{'Date Published : ' + (selectedVideo as any).snippet.publishedAt}</p>
+                        <p>
+                          {!readMore && descPreview}
+                          {readMore && extraContent}
+                          <Button
+                            icon
+                            labelPosition={readMore ? 'left' : 'right'}
+                            color="teal"
+                            floated="right"
+                            size="mini"
+                            onClick={() => {
+                              setReadMore(!readMore);
+                            }}
+                          >
+                            {moreButtonName}
+                            <Icon className={readMore ? 'left arrow' : 'right arrow'} />
+                          </Button>
```

Comment:
This is a very nice improvement.

The functionality is very nice. I'm wondering if there might be a simpler way to represent the buttons. Something like how youtube displays them would be nice. Ex:

<img width="209" alt="Screen Shot 2021-01-23 at 4 58 53 PM" src="https://user-images.githubusercontent.com/12631935/105619248-6cb94400-5da5-11eb-8ab3-e40c1adb70f0.png">

<img width="148" alt="Screen Shot 2021-01-23 at 4 59 04 PM" src="https://user-images.githubusercontent.com/12631935/105619252-717df800-5da5-11eb-8f5e-55ffff570512.png">

I like these because they aren't very "button like" so they are more subtle, while still being very clear. They also fit nicely right below the text. What do you think?

Reply:
There is no built-in ui item that matches the "show more" style. So for now, I can not do it without coding a totally new component or CSS class, which is not necessary I guess.